internal/worker/logsink/manifold.go

```diff
@@ -88,11 +97,17 @@ func Manifold(config ManifoldConfig) dependency.Manifold {
 				return nil, errors.Trace(err)
 			}
 
-			var controllerDomainServices services.ControllerDomainServices
-			if err := getter.Get(config.DomainServicesName, &controllerDomainServices); err != nil {
+			var logSinkServicesGetter services.LogSinkServicesGetter
+			if err := getter.Get(config.LogSinkServices, &logSinkServicesGetter); err != nil {
 				return nil, errors.Trace(err)
 			}
-			controllerCfg, err := controllerDomainServices.ControllerConfig().ControllerConfig(ctx)
+
+			var controllerService services.ControllerLogSinkServices
+			if err := getter.Get(config.LogSinkServices, &controllerService); err != nil {
+				return nil, errors.Trace(err)
+			}
```

Comment:
Why separate the controllerServices off into another worker like this?

Reply:
Because you can't get the controller services without the model UUID, and because we don't know the modelUUID until later on, we need to get the controller services directly.